api/app/controllers/spree/api/v2/storefront/products_controller.rb

```diff
@@ -42,6 +42,10 @@ def model_class
           def scope_includes
             product_list_includes
           end
+
+          def allowed_sort_attributes
```

Comment:
@szymoniwacz let's also add this change to the platform api as well - https://github.com/spree/spree/blob/master/api/app/controllers/spree/api/v2/platform/products_controller.rb